app/assets/javascripts/initializers/initializeVideoPlayback.js

```diff
@@ -136,12 +136,22 @@ function initializeVideoPlayback() {
       return;
     }
 
-    if (message.action == 'pause') {
-      getById('pause-butt').classList.remove('active');
-      getById('play-butt').classList.add('active');
-      videoPlayerEvent(false);
-    } else if (message.action == 'tick') {
-      currentTime = message.currentTime;
+    switch (message.action) {
```

Comment:
Adding more events here would just made this `if else` uglier so I made the "switch" (pun intended)